src/main/resources/application-test.yml

```
@@ -0,0 +1,6 @@
+spring:
+  datasource:
+    url: jdbc:h2:mem:db_test;MODE=MySQL
+    username: sa
+    password:
+    driver-class-name: org.h2.Driver
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)